test/e2e/data/ci-artifacts-platform-kustomization.yaml

```diff
@@ -9,3 +9,25 @@ metadata:
   name: ${CLUSTER_NAME}-control-plane
 spec:
   version: ${KUBERNETES_VERSION}
+# Comment this out for debbuging in case there are problems with the ssky key
```

Comment:
Is it `ssh` key?

Reply:
The idea is to use the commented out section if there is a problem with the OpenStack keypair which is otherwise used in the e2e tests. Because if there is a problem without it's very hard to debug because you then cannot ssh onto the nodes.

Reply:
> "Problems with the ssky key"

I meant is it "ssh key" instead of ssky key

Reply:
Ups, yes that's a typo :)

Reply:
I changed it a bit to mention the OpenStack keypair mechanism explicitly

Reply:
Thanks !!